Makefile

```diff
@@ -26,10 +26,12 @@ OBJS =  $(patsubst %.f,%.f.o,\
 	$(patsubst %.S,%.S.o,\
 	$(patsubst %.c,%.c.o,$(filter-out $(addprefix src/,$(DUPLICATE_SRCS)),$(SRCS)))))
 
-# If we're on windows, don't do versioned shared libraries.  If we're on OSX,
-# put the version number before the .dylib.  Otherwise, put it after.
+# If we're on windows, don't do versioned shared libraries. Also, generate a link library
```

Comment:
isn't this generally called an import library? https://msdn.microsoft.com/en-us/library/windows/desktop/ms682592(v=vs.85).aspx